### PR TITLE
Bug/searchbar autocomplete

### PR DESCRIPTION
**Pull request by OscarLarm**
### Description
This pull fixes the bug where elements shift on some viewports when searching for only one card. I set a minimum height for the card container, and I put the searchbar in a seperate search container.

### Details
- Fix shifting elements when searching
- Put the searchbar in a seperate container for easier flex

### References

<br>

### CODE REVIEW

**Comment by OscarLarm** [comment]
The merge issues should be fixed now

File: interface/templates/index.html [hunk 1]
```diff
@@ -26,12 +26,16 @@ <h1 id="landingHeaderTitle">
...
+            <div id="searchContainer">
+                <input id="search" type="text" placeholder="Search...">
+            </div>
+
             <div id="cardHolder">
                 <!--cards from 'generate-card.js goes here'-->
             </div>
         </div>
-        <button id="myBtn" title="Go to top">Top</button>
+        <button onclick="topFunction()" id="myBtn" title="Go to top">Top</button>
```
Comment (annk15) [inline interface/templates/index.html]:
```suggestion
        <button id="myBtn" title="Go to top">Top</button>
```

**Review by annk15 — CHANGES_REQUESTED** [review]
Good work! It does however look like one of the old lines for the scroll button has been committed so it should be reverted before merging this.

**Review by annk15 — APPROVED** [review]
Everything looks good now, I will merge!

**Pull request merged**